### PR TITLE
Trouble shooting guide

### PR DESCRIPTION
**Pull request by uablrek**
Add a Meridio Troubleshooting Guide.

### CODE REVIEW

File: docs/trouble-shooting/README.md [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+kubectl exec -n red meridio-proxy-96nvc -- ifconfig
+# Or
+kubectl exec -it -n red meridio-load-balancer-5785674d6-4hqsh -c load-balancer -- sh
+```
+
+We will give examples how to troubleshoot individual Meridio
+containers below.
+
+
+### Ephemeral containers
```
Comment (LionelJouin) [inline docs/trouble-shooting/README.md:151]: This feature is not available before Kubernetes 1.23 right?

File: docs/trouble-shooting/README.md [hunk 2]
```diff
@@ -0,0 +1,342 @@
...
+
+--- 172.16.1.8 ping statistics ---
+1 packets transmitted, 1 received, 0% packet loss, time 0ms
+rtt min/avg/max/mdev = 0.859/0.859/0.859/0.000 ms
+```
+
+Traffic to the target goes via a "proxy".
+
+
+## Proxy
```
Comment (LionelJouin) [inline docs/trouble-shooting/README.md:298]: The source routes have to be verified in the proxy. it should contains 1 rule per VIP with a table assigned. And in each table there should be nexthops with the IPs of the load-balancers

File: docs/trouble-shooting/README.md [hunk 3]
```diff
@@ -0,0 +1,342 @@
...
+(fwmark) used to route traffic to different target PODs;
+
+```
+# kubectl exec -n red meridio-load-balancer-5785674d6-6cqhm -c load-balancer -- ip rule
+0:      from all lookup local
+96:     from all fwmark 0x5a lookup 90
+97:     from all fwmark 0x43 lookup 67
+98:     from all fwmark 0x3a lookup 58
+99:     from all fwmark 0x30 lookup 48
+100:    from 10.0.0.1 lookup 4096
```
Comment (LionelJouin) [inline docs/trouble-shooting/README.md:273]: maybe This rule could be explained:
`100: from 10.0.0.1 lookup 4096`

File: docs/trouble-shooting/README.md (hunk 2 above)
Comment (uablrek) [reply docs/trouble-shooting/README.md:298]: Will add those

**Pull request merged**